src/App.js

```diff
@@ -1,25 +1,156 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.scss";
 
-function App() {
+import ButtonB from "react-bootstrap/Button";
+import Table from "react-bootstrap/Table";
+import Form from "react-bootstrap/Form";
+import Modal from "react-bootstrap/Modal";
+import React from "react";
+
+const App = () => {
+  const [show, setShow] = React.useState(false);
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+  const [input, setInput] = React.useState({
+    name: "",
+    lastname: "",
+  });
+  const persons = [
+    {
+      id: 1,
+      name: "Jose",
+      lastname: "Alvarez",
+      gender: "male",
+      address: "Av. Don Bosco",
+      birthday: "18/03/1994",
+    },
+    {
+      id: 2,
+      name: "Jose",
+      lastname: "Antonio",
+      gender: "male",
+      address: "Calle Guayas",
+      birthday: "26/08/1989",
+    },
+  ];
+  
+  const addPerson = async (e) => {
+    handleClose();
+     persons.push({
+      id: persons.length+1,
+      name: input.name,
+      lastname: input.lastname,
+      gender: "male",
+      address: "Calle Guayas",
+      birthday: "26/08/1989",
+    });
+    setInput({
+      name: "",
+      lastname: "",
+      })
+  };
+
+  
+  const handleInputChange =(e)=>{
+    setInput({
+      ...input,
+      [e.target.name]: e.target.value,
+      })
+    }
+  
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
+    <>
+      <div className="container">
+        <div className="container">
+          <h2>
+            Ajax CRUD with bootstrap modals and Datatables with Bulk Delete
```

Comment:
eso es lo que estas haciendo? creo que el copy and paste fue muy literal xD

Reply:
eso es lo que dice en la imagen que me mandaste jajaja